packages/radix-vue/src/Tooltip/TooltipContent.vue

```diff
@@ -23,12 +24,13 @@ const { forwardRef } = useForwardExpose()
 </script>
 
 <template>
-  <component
-    :is="rootContext.disableHoverableContent.value ? TooltipContentImpl : TooltipContentHoverable"
-    v-if="rootContext.open.value"
-    :ref="forwardRef"
-    v-bind="forwarded"
-  >
-    <slot />
-  </component>
+  <Presence :present="rootContext.open.value">
```

Comment:
For `Presence` component usage we should also add a new prop `forceMount`, and set it as such. 

```suggestion
  <Presence :present="forceMount || rootContext.open.value">
```

Reply:
I added forceMount to TooltipContent 👍